tests/setops_test.py

```diff
@@ -4,76 +4,69 @@
 from context import arkouda as ak
 from base_test import ArkoudaTest
 
-if __name__ == "__main__":
-    import argparse, sys
-    parser = argparse.ArgumentParser(description="Runs and times reductions over arrays in both arkouda and numpy.")
-    parser.add_argument('hostname', help='Hostname of arkouda server')
-    parser.add_argument('port', type=int, help='Port of arkouda server')
-    parser.add_argument('-n', '--size', type=int, default=10**6, help='Problem size: length of array to argsort')
+SIZE = 1000
+OPS = frozenset(['intersect1d', 'union1d', 'setxor1d', 'setdiff1d'])
 
-    args = parser.parse_args()
-    ak.verbose = False
-    ak.connect(args.hostname, args.port)
-    print("size = ",args.size)
-    SIZE = args.size
-    a = ak.randint(0, 2*SIZE, SIZE)
-    b = ak.randint(0, 2*SIZE, SIZE)
+def make_arrays():
+    a = ak.randint(0, SIZE, SIZE)
+    b = ak.randint(SIZE/2, 2*SIZE, SIZE)
+    return a, b
 
+def compare_results(akvals, npvals) -> int:
+    '''
+    Compares the numpy and arkouda arrays via the numpy.allclose method with the
+    default relative and absolute tolerances, returning 0 if the arrays are similar
+    element-wise within the tolerances, 1 if they are dissimilar.element
     
-    set_union = ak.union1d(a,b)
-    print("union1d = ", set_union.size,set_union)
-    # elements in a or elements in b (or in both a and b)
-    passed = ak.all(ak.in1d(set_union,a) | ak.in1d(set_union,b))
-    print("union1d passed test: ",passed)
+    :return: 0 (identical) or 1 (dissimilar)
+    :rtype: int
+    '''
+    akvals = akvals.to_ndarray()
     
-    set_intersection = ak.intersect1d(a,b)
-    print("intersect1d = ", set_intersection.size,set_intersection)
-    # elements in a and elements in b (elements in both a and b)
-    passed = ak.all(ak.in1d(set_intersection,a) & ak.in1d(set_intersection,b))
-    print("intersect1d passed test: ",passed)
+    if not np.array_equal(akvals,npvals):
+        print(f"Different values (abs diff = {np.abs(akvals - npvals).sum()}")
```

Comment:
this computation is not right for this kind of test. you should print the mismatches between results. we used in1d to test the differences in the original code maybe think about how to apply that

Reply:
Changed that to hopefully more what you were talking about, where it prints the values that are in the np array, but not the ak array and also the other way around. Let me know if you had something else in mind. Also, it looks like I am not able to tag reviewers myself. After some research, I think I have to be marked as a "collaborator" to be able to request a review